tests/pivots/spreadsheet_pivot/spreadsheet_pivot.test.ts

```diff
@@ -1808,6 +1808,27 @@ describe("Spreadsheet Pivot", () => {
     expect(getEvaluatedCell(model, "B27").formattedValue).toBe("2022/04/14 01:02:03");
   });
 
+  test("Pivot aggregates duration fields with correct SUM and AVG values", () => {
+    const model = createModelFromGrid({
+      A1: "Duration",
+      A2: "01:30:00",
+      A3: "02:00:00",
+      A4: "00:45:00",
+    });
+    addPivot(model, "A1:A4", {
+      columns: [],
+      rows: [],
+      measures: [
+        { id: "Duration:avg", fieldName: "Duration", aggregator: "avg" },
+        { id: "Duration:sum", fieldName: "Duration", aggregator: "sum" },
+      ],
+    });
+    setCellContent(model, "A6", '=PIVOT.VALUE(1,"Duration:avg")');
+    setCellContent(model, "B6", '=PIVOT.VALUE(1,"Duration:sum")');
+    expect(getEvaluatedCell(model, "A6").formattedValue).toBe("01:25:00");
+    expect(getEvaluatedCell(model, "B6").formattedValue).toBe("04:15:00");
```

Comment:
nitpick: you could also add a expect to check that the field in pivot.getFields() has the correct type